src/System.Net.Http/src/System/Net/Http/HttpResponseMessage.cs

```diff
@@ -198,6 +198,12 @@ public override string ToString()
             sb.Append(", Headers:\r\n");
             HeaderUtilities.DumpHeaders(sb, _headers, _content?.Headers);
 
+            if (_trailingHeaders != null)
+            {
+                sb.Append(", Trailing Headers:\r\n");
```

Comment:
Any particular reason we use \r\n even on Unix?

Reply:
It's part of the protocol specification.

Reply:
\r\n is part of the HTTP protocol itself.  But the ToString() method here isn't used by the protocol.  It's simply used as diagnostics.  But all of the ToString() methods in HttpClient and related classes came from .NET Framework which only ran on Windows. So, \r\n was always used.

We could convert all \r\n things in all the ToString() methods to use something like `Environment.NewLine`.  But it's seems easier now to just keep things consistent with all the rest of the \r\n being used in ToString() methods.

Reply:
I debated this when I saw it as part of making this change, and I meant to comment on it but forgot to after putting up the PR.  I do think it's something we should revisit, and I'll open an issue for it, but I don't want to change it as part of this PR, which should remain consistent as David says with all other uses.